src/app/shared/components/SelectGrid/GridItem/index.tsx

```diff
@@ -0,0 +1,56 @@
+'use client'
+
+import clsx from 'clsx'
+
+import { usePathname } from 'next/navigation'
+
+import Image from "next/image"
+import Link from "next/link"
+
+type GridItemProps = {
+  entityLogo?: object,
+  entity: {
+    key?: string,
+    primaryColor?: string,
+    title: string,
+  },
+  hideTitle?: boolean,
+}
+
+const GridItem = (props: GridItemProps) => {
+  const { entityLogo, entity, hideTitle } = props
+  const { src } = entityLogo || {}
+  const { key, primaryColor, title } = entity
+
+  const pathname = usePathname()
+
+  // WIP the backgroundColor here on line 35 isn't working because tailwind doesn't handle dynamic class names
+  // https://tailwindcss.com/docs/content-configuration#dynamic-class-names
```

Comment:
following this doc, we have to construct full tailwind class strings, we can't interpolate data, so we can do something like this

create a function to return the full strings and then call it

```
const getBackgroundColorString = (colour: string) => {
  const colourMap = {
    '#43B3E6': 'bg-[#43B3E6]',
    '#000000': 'bg-[#000000]',
    '#5E0224': 'bg-[#5E0224]',
    ...other colours here
  }

  return colourMap[colour]
}
```

BUT im not even sure we should bother adding a background colour here, it looks good with white background anyway, i would remove it

Reply:
Yeh I thought it might look better with the whole tile's having the team colour and maybe making them a bit smaller. But have left for now and removed